meta/menatwork/contao-multicolumnwizard-bundle/de.yml

```diff
@@ -0,0 +1,11 @@
+de:
+    title: MultiColumnWizard
+    description: >
+        Der MultiColumnWizard ist ein Widget zur Abbildung von mehreren Feldern gleichen und/oder unterschiedlichen Typs (Inputtype) in einem DCA-Element.
+
+        Mehr unter http://de.contaowiki.org/MultiColumnWizard
+    keywords:
+        - contao
+        - bundle
+        - widget
```

Comment:
what do you think about adding „mcw“ here? :)

Reply:
fixed, thx